modules/jpa-dao/src/test/java/com/devonfw/example/general/common/api/config/BeansOrikaConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.devonfw.example.general.common.api.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import ma.glasnost.orika.MapperFacade;
+import ma.glasnost.orika.MapperFactory;
+import ma.glasnost.orika.impl.DefaultMapperFactory;
+
+/**
+ * Java bean configuration for Dozer
+ */
+@Configuration
+@ComponentScan(basePackages = { "com.devonfw.module.beanmapping" })
+public class BeansOrikaConfig {
+
+  @Bean
+  public MapperFacade getOrika() {
+
+    MapperFactory factory = new DefaultMapperFactory.Builder().build();
+
+    MapperFacade orika = factory.getMapperFacade();
```

Comment:
This is insufficient. We need to configure the Mapping from Entity to ETO. This was done for dozer via some magic in the XML. For orika this is already solved in the JUnit test for orika.
My suggestion would be to use the same approach as for Jackson:
* In `devon4j-beanmapping-orika` we provide a base config class that configures cross-cutting stuff such as this Entity to ETO mapping.
* Our `BeansOrikaConfig` class in app-template derives from this base config class and is the place to add project specific custom mappings and even allows to override/disable/modify the defaults from the base config class.